lib/controllers/ServiceFabrikAdminController.js

```diff
@@ -623,6 +601,75 @@ class ServiceFabrikAdminController extends BaseController {
         .send({}));
   }
 
+  startMaintenance(req, res) {
+    logger.info('req.body=>', req.body);
+    return maintenanceManager
+      .getMaintenaceInfo()
+      .then(body => {
+        if (body === null) {
+          return maintenanceManager
+            .startMaintenace(req.body, req.user)
+            .then(body => res
+              .status(201)
+              .send(body));
+        } else {
+          res
+            .status(403)
+            .send({
+              errorMessage: 'System already in maintenance mode',
+              maintenanceInfo: body
+            });
+        }
+      });
+  }
+
+  updateMaintenance(req, res) {
+    return maintenanceManager
+      .getMaintenaceInfo()
+      .then(body => {
+        if (body !== null) {
+          return maintenanceManager
+            .updateMaintenace(req.body.progress, req.body.state, req.user)
+            .then(body => res
+              .status(200)
+              .send(body));
+        } else {
+          res
+            .status(403)
+            .send({
+              errorMessage: 'System not in maintenance mode'
+            });
+        }
+      });
+
+  }
+
+  getMaintenance(req, res) {
+    return maintenanceManager
+      .getMaintenaceInfo()
+      .then(body => {
+        if (body === null) {
+          res
+            .status(404)
+            .send({
+              system_in_maintenance: false
+            });
+        } else {
+          _.set(body, 'system_in_maintenance', true);
```

Comment:
is it better to put the value 'system_in_maintenance' in constant ?

Reply:
As discussed these are coding standards that exist all across code base. Also these attribs are used at one off place. So no need to really parameterize them.